src/metrics.rs

```diff
@@ -0,0 +1,71 @@
+use lazy_static::lazy_static;
+use prometheus::{Encoder, Histogram, HistogramOpts, IntCounterVec, Opts, Registry};
+
+use crate::error::Error;
+
+lazy_static! {
+    pub static ref REGISTRY: Registry = Registry::new();
+    pub static ref PROCESSED_MAIL_REQUESTS: IntCounterVec = IntCounterVec::new(
+        Opts::new("processed_mail_requests", "Processed Mail Requests"),
+        &["status"]
+    )
+    .unwrap_or_else(|e| {
+        tracing::error!(
+            error = e.to_string(),
+            "processed_mail_requests metric error"
+        );
+        std::process::exit(1);
+    });
+    pub static ref MAIL_TESTS: IntCounterVec =
+        IntCounterVec::new(Opts::new("mail_tests", "Mail Tests"), &["status"]).unwrap_or_else(
+            |e| {
+                tracing::error!(error = e.to_string(), "mail_tests metric error");
+                std::process::exit(1);
+            }
+        );
+    pub static ref PROCESSING_TIME: Histogram = Histogram::with_opts(HistogramOpts {
+        common_opts: Opts::new("processing_time", "Mail Processing Times"),
```

Comment:
```suggestion
        common_opts: Opts::new("mail_test_processing_time", "Mail Processing Times"),
```

Reply:
Dette er processing time for vanlig epost, ikke testen.